lib/toolbox/args.c

```diff
@@ -34,6 +34,25 @@ bool args_read_int_and_trim(FuriString* args, int* value) {
     return false;
 }
 
+bool args_read_float_and_trim(FuriString* args, float* value) {
+    size_t cmd_length = args_get_first_word_length(args);
+    if(cmd_length == 0) {
+        return false;
+    }
+
+    // TODO-nofl change to float
+    int32_t temp;
```

Comment:
@skotopes for now there are now way to extract float?

Reply:
nope ) feel free to implement

Reply:
I found `strtof`, it seems to work